Makefile

```diff
@@ -26,5 +26,6 @@ verify-podnetworkconnectivitychecks:
 test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
 .PHONY: test-e2e-encryption
 
-test-e2e-monitoring:
+test-e2e-monitoring: GO_TEST_PACKAGES :=./test/e2e-monitoring/...
```

Comment:
I think it would be useful to add verbosity in case the test starts failing in CI in the future.
You can do that by adding:
```
test-e2e-encryption: GO_TEST_FLAGS += -v
```

Reply:
I've added similar flags in addition to this, however, one that seemed quite off was the 4 hour timeout limit for the `encryption` test. Is that taking into account certain build overheads (which only happen in the CI)? I've added a 5 minute timeout for the `monitoring` test, but LMK if that should be increased to a similar amount.